event.go

```diff
@@ -215,10 +304,14 @@ const (
 	Software
 )
 
+type Payload interface {
+	// Define common methods for all payloads here, if any.
+}
```

Comment:
I'm not sure why there is an empty struct here.

Reply:
Because of [setFeedback](https://docs.elgato.com/sdk/plugins/events-sent#setfeedback-sd) event. It can contain any key. And I do not see any other option in that case without code duplication.